roxctl/deployment/check/check.go

```diff
@@ -116,6 +117,13 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c.Flags().BoolVar(&deploymentCheckCmd.force, "force", false, "bypass Central's cache for images and force a new pull from the Scanner")
 	utils.Must(c.MarkFlagRequired("file"))
 	c.Flags().StringVar(&deploymentCheckCmd.cluster, "cluster", "", "cluster name or ID to use as context for evaluation")
+	c.Flags().StringVarP(&deploymentCheckCmd.namespace, "namespace", "n", "default", "namespace to evaluate, defaults to \"default\" if unset.")
+	namespace, err := c.Flags().GetString("namespace")
+	// this error should never occur, it would only occur if default values are invalid
+	utils.Must(err)
+	if namespace == "default" {
+		fmt.Printf("Warning: The namespace defaulted to \"default\", if you want to check deployments in a custom namespace set it with '%s [X]' or '-%s [X]'\n", c.Flag("namespace").DefValue, c.Flag("namespace").Shorthand)
```

Comment:
Do we have another option in `roxctl` to orient the phrasing of that help text to? I don't think I've seen `'%s [X]' or '-%s [X]` as a phrase somewhere else, but please correct me if I'm wrong 😄 

That said, follow up question for this new flag:
Do we require the `Cluster` to be set when providing the `Namespace` flag? If not, should we do that? IIRC, Namespaces only have to be unique in single a Cluster, making it possible to have multiple namespaces with the same name, right?

Reply:
I'll hardcode `--namespace` and `-n` into the string, less confusing.

Another warning when `Cluster` is not set but `Namespace` is seems appropriate, I'll add one.